backend/app/services/wage_chunk_formatting_service.rb

```diff
@@ -17,7 +17,7 @@ def pay_period_description
                 start_date = I18n.l chunk[:start_date].to_date, format: :long
                 end_date = I18n.l chunk[:end_date].to_date, format: :long
                 hours = chunk[:hours]
-                rate = chunk[:rate]
+                rate = '%.2f' % chunk[:rate]
```

Comment:
Just to clarify, this is to display money with 2 decimals in the PDFs?

Reply:
Yes. It does work but I don't understand why Ruby Formatting tests fail.

Reply:
Look at your error from Rubocop. You can google it to see how the code should be modified.